pkg/deployments/tasks/deprecated/20210418-weighted-pool/readme.md

```diff
@@ -0,0 +1,12 @@
+# 2021-04-18 Weighted Pool
+
+Deployment of the `WeightedPoolFactory`, for Weighted Pools of up to 8 tokens, and `WeightedPool2TokensFactory`, for Weighted Pools restricted to 2 tokens, but with an optional price oracle.
```

Comment:
Let's add a comment here saying this is deprecated and linking to the new task (see e.g. the old stable pool readme for a reference)